src/storage/txn/flow_controller/tablet_flow_controller.rs

```diff
@@ -117,13 +117,7 @@ impl FlowInfoDispatcher {
                     }
 
                     let insert_limiter_and_checker = |region_id, suffix| -> FlowChecker<E> {
-                        let engine = tablet_factory
-                            .open_tablet(
-                                region_id,
-                                Some(suffix),
-                                OpenOptions::default().set_cache_only(true),
-                            )
-                            .unwrap();
+                        let engine = registry.get(region_id).unwrap().latest().unwrap().clone();
```

Comment:
Weak reference would be handy here. Maybe leave a todo.

Reply:
I think this part needs to be refactored. Flow control should be bound to the lifetime of a peer instead of tablets. The refactor will be done later.